### ISSUE TITLE
Grundig GDSP5470DXSCH extractor hood supported?

### ISSUE THREAD

**Issue by bjacobse** [home-assistant-HomeWhiz/home-assistant-HomeWhiz#219]
I guess this extractor hood is not yet supported?
in my Android app, I can use the homewhiz app via wifi to control the extractor hood.
I have managed to add this via the cloud connection to home assistant, but as expected the buttons are greyed out as this device isn't supported.

I am using proxmox and have installed home assistant via the helperscript (I think it is called)

How do I go further?

![extractor_hood](https://github.com/home-assistant-HomeWhiz/home-assistant-HomeWhiz/assets/4013602/9113445c-ae43-436c-8855-bf90cb7cbd0e)
[home-assistant_homewhiz_2024-05-23T21-00-09.045Z.log](https://github.com/home-assistant-HomeWhiz/home-assistant-HomeWhiz/files/15422644/home-assistant_homewhiz_2024-05-23T21-00-09.045Z.log)

**Comment by TechHummel** [comment]
Thanks for bringing this up. While I am not sure whether extractor hoods are supported, this could also be related to #216 .

**Comment by TechHummel** [comment]
Can you try the latest beta version and see if anything has changed? Thanks!

**Comment by bjacobse** [comment]
Thank you very much.
I downloaded version 0.0.6
Deleted my old extraction fan, because I had forgotten the password.
In the app for the phone, i clicked forgot password.
On computer I clicked password link. And created new password.
Use app to connect with new password, works.
In home assistant in Homewhiz i connet, email+password. Device gets conneted and installed.

Hurra i can switch light on/off
I can switch fan on and change level, and switch off.
Thank you my little test succes
